src/engine/runtime.js

```diff
@@ -1902,6 +1910,14 @@ class Runtime extends EventEmitter {
         return this._cloneCounter < Runtime.MAX_CLONES;
     }
 
+    /**
+     * Report that targets have finished being installed in the Virtual Machine.
+     * @param {Target} installedTargets - the newly installed targets.
+     */
+    emitTargetsInstalled (installedTargets) {
```

Comment:
Is there a reason that this event lives in the runtime instead of the VM?

Reply:
(This might be the right place for it to live, I just wanted to know what the reasoning for it living here is)

Reply:
Good Q! I wanted this event to be available to the extensions (ie listenable in `scratch3_video_sensing/index.js`), and they are already able to listen to runtime events. From what I can tell, currently they're not able to listen to VM events, but let me know if that's not the case.